cli/tests/e2e/snapshots/test_output/test_yaml_metavariables/report.json

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
those e2e tests are not super readable. Who is gonna look at the value on those expected report.json?

Reply:
I've read them all, and verified they look good. As long as they don't change, from now on, then we should be good.